TrafficCapture/dockerSolution/src/main/docker/migrationConsole/console_api/console_api/apps/orchestrator/urls.py

```diff
@@ -6,4 +6,5 @@
     path('osi-start-migration', views.osi_start_migration, name='osi-start-migration'),
     path('osi-stop-migration', views.osi_stop_migration, name='osi-stop-migration'),
     path('osi-delete-migration', views.osi_delete_migration, name='osi-delete-migration'),
+    path('osi-get-status-migration', views.osi_get_status_migration, name='osi-get-status-migration'),
```

Comment:
Do we test these API calls within integ tests? I'm concerned about the assumptions about response structure from OSI.

Reply:
We currently only have unit tests around this, with no integ tests